examples/example-node-sled-lib.js

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env node
+const assert = require('assert');
+const cp = require('child_process');
+const fs = require('fs');
+
+// Set ENV=testing so we don't make real requests to AWS APIs
+const env = Object.create(process.env);
+env.ENV = 'testing';
```

Comment:
There's a reasonable chance that something else could be setting or relying on an environment variable called "ENV".  Could we prefix this with something?  "CUMULUS_ENV" or similar?

Reply:
Addressed in https://github.com/cumulus-nasa/cumulus-sled/pull/2/commits/b24b1d1614eade0a199f159b11f619bc5caefbe1